variables.tf

```diff
@@ -233,3 +233,15 @@ variable "ip_address_timeout" {
   type        = string
   default     = "5m"
 }
+
+variable "nginx_controller" {
+  description = "Whether to have nginx controller installed in this cluster setup; with a dedicated IP"
```

Comment:
```suggestion
  description = "Whether to have a NGINX Ingress Controller installed in this cluster. Refer to the IP name to be used for the ingress IP"
```

Reply:
Maybe this would be better. 
`Whether to have a NGINX Ingress Controller installed in this cluster; with a dedicated IP. Refer to the IP name in nginx_ip_names to be used here.`
The reason i don't want to specify ingress IP is to avoid people mistakenly using the IP name from the `ingress_ip_names` which provides global IP, while only regional IP will work with this NGINX Ingress.

Reply:
Fair point. Let's refer to it as `\"var.nginx_ip_names\"` then in the var description to be more clear about where they should look for it.

---

This bring me to another concern that i have been contemplating. Maybe we should revise the way we declare IP names for different purposes with different global vs regional settings. The current way is a bit non-intuitive and sometimes hard to understand the expectations. Refactoring this will be an effort for later though. Just sharing about what's coming up.

Reply:
My suggestion is, we should ask ourselves this question- if over the period of time, we need more IPs for system X, Y, Z etc. will we end up saying X_ip_names, Y_ip_names etc.?
Or, should we rather have two arrays, calling `regional_ip_names` `global_ip_names`, and under which for specific system we have proper prefixes- `x-ip`, `nginx-lb-ip` etc.

Reply:
Think from generic perspective, not too specific.

Reply:
Anyway since there is a revision coming in terms on how to declare IP name. I believe the specific IP names will be removed. So let's go with `ip_names`.

Reply:
Sounds good